snuba/snuba_migrations/spans_v2/0001_spans_v2.py

```diff
@@ -0,0 +1,190 @@
+from typing import Sequence
+
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+from snuba.utils.schemas import (
+    UUID,
+    Bool,
+    Column,
+    DateTime,
+    DateTime64,
+    Float,
+    Int,
+    Map,
+    String,
+    UInt,
+)
+
+storage_set_name = StorageSetKey.SPANS_V2
+local_table_name = "spans_v2_local"
+dist_table_name = "spans_v2_dist"
+
+columns: Sequence[Column[Modifiers]] = (
+    [
+        Column("organization_id", UInt(64)),
+        Column("project_id", UInt(64)),
+        Column("trace_id", UUID()),
+        Column("span_id", UInt(64)),
+        Column("parent_span_id", UInt(64, Modifiers(codecs=["ZSTD(1)"]))),
+        Column("segment_id", UInt(64, Modifiers(codecs=["ZSTD(1)"]))),
+        Column("segment_name", String(Modifiers(default="''", codecs=["ZSTD(1)"]))),
```

Comment:
I don't think we should have a default here. The Rust struct will specify one anyway and we don't have default for other columns, so either we have default everywhere or not at all (I think not at all is better).